fasta/multiFa.go

```diff
@@ -166,3 +166,89 @@ func ChangePrefix(records []Fasta, prefix string) {
 		records[idx].Name = fmt.Sprintf("%s_%d", prefix, idx)
 	}
 }
+
+//PairwiseMutationDistanceReferenceWindow takes two input fasta sequences and calculates the number of mutations in a reference window of a given size. Segregating sites are counted as 1, as are INDELs regardless of length.
+//alnStart indicates the beginning alignment column for distance evaluation, and windowSize is the number of references bases to compare.
+//Three returns, first is the pairwise mutation distance, second is reachedEnd, a bool that is true for incomplete windows. The third return is alignmentEnd, or the last alignment column evaluated.
+func PairwiseMutationDistanceReferenceWindow(seq1 Fasta, seq2 Fasta, alnStart int, windowSize int) (int, bool, int) {
+	diff := 0
+	baseCount := 0
+	var seq1Indel bool = false
+	var seq2Indel bool = false
+	var reachedEnd bool = false
+	var i int = 0
+
+	for i = alnStart; baseCount < windowSize && i < len(seq1.Seq); i++ {
+		if seq1.Seq[i] == seq2.Seq[i] {
+			if seq1.Seq[i] != dna.Gap {
+				seq1Indel = false
+				seq2Indel = false
+				baseCount++
+			}
+		} else if seq1.Seq[i] == dna.Gap {
+			seq2Indel = false
+			if !seq1Indel {
+				seq1Indel = true
```

Comment:
This is not related to the current PR but I was wondering if there's a way to keep track of indel sizes and use them in programs like multiFaAcceleration, like maybe in the stage where velocity gets normalized, a bigger indel size can become a multiplier that gets multiplied to the velocity (but not initial velocity) and creates a bigger acceleration (when we calculate a=v-v0)?